brave/brave-core
# Issues/23978

Added triggering of the notification timed-out event when custom ad notifications are disabled.


Resolves https://github.com/brave/brave-browser/issues/23978

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

For Desktop:
- Trigger Native Brave ad notification  on one of the desktop platforms (Windows, MacOS, Linux)
- Wait until the ad notification is closed by timeout
- Check that the time-out event was triggered (`Timed out notification ad with placement id ` message in rewards-log)

For Android:
- Check that Brave ad notification is shown on Android
